lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
+ *
+ * Type: `Boolean`
+ *
+ * Value: `true`
+ *
+ * Version: `ES6`
+ *
+ * #### Example
+ *
+ * ```js
+ * "requireShorthandArrowFunctions": true
+ * ```
+ *
+ * ##### Valid
+ *
+ * ```js
+ * evens.map(v => v + 1);
+ * evens.map(v => {
+ *     v = v + 1;
```

Comment:
I wasn't sure if I needed this example but it's just to show it doesn't error when the function has multiple statements in it since you need the braces and return in that case.